UKHomeOffice/sgmr-service
# NMSW-771 Restrict field length on place of birth field

### AC / Description of changes

Error reported : The 'PlaceOfBirth' element is invalid - The value 'Z IJsselmeerpolders, Ther Netherlands' is invalid according to its datatype 'T_PlaceOfBirth' - The actual length is greater than the MaxLength value.

### To Test

- run app
- go to people
- edit a person
- > you should not be able to enter more than 35 characters in place of birth
- try pasting a longer word(s) in the field
- > it should truncate at 35 characters

### Notes

---

**Remember to merge to feature branches not `master`.**
